hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestFailureHandlingByClient.java

```diff
@@ -70,26 +70,32 @@
 import org.apache.hadoop.ozone.om.helpers.OmKeyInfo;
 import org.apache.hadoop.ozone.om.helpers.OmKeyLocationInfo;
 import org.apache.ratis.proto.RaftProtos;
-import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.EnumSource;
 
 /**
  * Tests Exception handling by Ozone Client.
  */
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class TestFailureHandlingByClient {
 
-  private MiniOzoneCluster cluster;
-  private OzoneConfiguration conf;
-  private OzoneClient client;
-  private ObjectStore objectStore;
-  private int chunkSize;
-  private int blockSize;
-  private String volumeName;
-  private String bucketName;
-  private String keyString;
-  private RaftProtos.ReplicationLevel watchType;
+  private static MiniOzoneCluster cluster;
+  private static OzoneConfiguration conf;
+  private static OzoneClient client;
+  private static ObjectStore objectStore;
+  private static int chunkSize;
+  private static int blockSize;
+  private static String volumeName;
+  private static String bucketName;
+  private static String keyString;
+  private static final List<DatanodeDetails> RESTART_DATA_NODES = new ArrayList<>();
```

Comment:
Instead of making all fields `static`, please add `@TestInstance(TestInstance.Lifecycle.PER_CLASS)` on the class.  This allows `@BeforeAll` and `@AfterAll` methods to be non-`static`, hence members can be non-`static`, too.

https://docs.junit.org/current/user-guide/#writing-tests-test-instance-lifecycle